tutorials/fundamentals/empty-bool.md

```diff
@@ -0,0 +1,29 @@
+There are only seven values that are [falsy](/tutorials/fundamentals/truthy) in JavaScript, and empty objects are not one of them.
+An empty object is an object that has no properties of its own.
+If you want to check if an object is empty you should use the `Object.keys()` function as it will return 0 on empty objects.
```

Comment:
Replace: "You can use the `Object.keys()` function to check if an object is empty as shown below."